include/kodlab_mjbots_sdk/mjbots_control_loop.h

```diff
@@ -61,7 +73,7 @@ class MjbotsControlLoop : public AbstractRealtimeObject {
   /*!
    * @brief function to be implemented by child. Must set torques in the robot class
    */
-  virtual void CalcTorques() = 0;
+  virtual void CalcTorques() {robot_->Update();}
```

Comment:
this interaction could change (as mentioned above)

Reply:
I think you should keep this default implementation (see other discussion in `robot_example.cpp`).

Reply:
I think we should not change this. Keep it as: `  virtual void CalcTorques() = 0;`

Reply:
I've come around to Shane's take so long as a `CalcTorques()` override is provided in `robot_example.cpp` that includes `robot_->Update()` and comments explaining its usage.

Reply:
Resolved in 060a657fd0508b266f5dde0c99b0d6b2a72ed7e7.